Dockerfile

```diff
@@ -64,6 +78,6 @@ CMD [ "-h" ]
 ## -- OpenNMS KARAF SSH   8101/TCP
 ## -- OpenNMS MQ         61616/TCP
 ## -- OpenNMS Eventd      5817/TCP
-## -- SNMP Trapd           162/UDP
-## -- Syslog Receiver      514/UDP
-EXPOSE 8980 18980 1099 8101 61616 5817 162/udp 514/udp
+## -- SNMP Trapd          1162/UDP
+## -- Syslog Receiver    10514/UDP
+EXPOSE 8980 1162
```

Comment:
As far I understand, everything here is exposed automatically by default, right? Should we expose Karaf Shell here as well by default?